src/__tests__/displayArticleTemplate.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import ArticleTemplate from '../Components/ArticleTemplate';
+
+describe('<ArticleTemplate />', () => {
+  const article = {
+    "id": 1,
+    "author": "Dash L.",
+    "date": "15th of August, 2019",
+    "title": "A study on the maladapted social behaviours of pidgeons",
+    "body": "They are seriously mean."
+  }
+  
+  it('renders the correct title', () => {
+    const describedComponent = shallow(<ArticleTemplate article={article} />)
+    const response = article.title
+    expect(describedComponent.contains(response)).toEqual(true)
+    //Code for troubleshooting: expect(describedComponent.text()).toEqual(response)
```

Comment:
Please remove comments in code. Add documentation to README

Reply:
Relocated comment to README for now as a note.  Will decide on whether to keep it in README later when considering document as a whole.